dat/data.base

```diff
@@ -503,18 +503,23 @@ bell of opening
 	* Old Tom was the single cracked bronze bell in the University
 	bell tower.
 		[ Eric, by Terry Pratchett ]
-*bell
-	They were old Chimes, trust me.  Centuries ago, these Bells had
-	been baptized by bishops: so many centuries ago, that the register
-	of their baptism was lost long, long before the memory of man, and
-	no one knew their names.  They had had their Godfathers and 
-	Godmothers, these Bells (for my own part, by the way, I would
-	rather incur the responsibility of being Godfather to a Bell than
-	a Boy), and had their silver mugs no doubt, besides.  
-	But Time had mowed down their sponsors, and Henry the Eighth had
-	melted down their mugs; and they now hung, nameless and mugless, 
-	in the church-tower.
-		[ The Chimes, by Charles Dickens ]
+~*engraved*bell*
```

Comment:
Vanilla doesn't have "engraved silver bell" as the unidentified description for Bell of Opening so this line isn't needed.